src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -658,4 +670,30 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         return email;
     }
 
+    public List<InstructorAttributes> generateCoOwnersList(String courseId) {
+        List<InstructorAttributes> instructors = instructorsLogic.getInstructorsForCourse(courseId);
+        List<InstructorAttributes> instructorsWithCoOwnerPrivileges = new ArrayList<InstructorAttributes>();
+        for (InstructorAttributes instructor : instructors) {
+            if (!instructor.hasCoownerPrivileges()) {
+                continue;
+            }
+            instructorsWithCoOwnerPrivileges.add(instructor);
+        }
+        return instructorsWithCoOwnerPrivileges;
+    }
+
+    private String generateCoOwnersEmailsLine(List<InstructorAttributes> coOwners) {
```

Comment:
Since this method is always called in the form of `generateCoOwnersEmailsLine(generateCoOwnersList(courseId))`, you could call `generateCoOwnersList` in this method instead.